features/signals_railway_signals.yaml

```diff
@@ -1838,6 +1842,44 @@ features:
       - { tag: 'railway:signal:combined', value: 'DE-BOStrab:h' }
       - { tag: 'railway:signal:combined:form', value: 'light' }
 
+  - description: Karlsruhe VBK Vorsignale
+    country: DE
+    icon:
+      match: 'railway:signal:distant:states'
+      cases:
+        - { regex: '^(.*;)?DE-VBK:fv1(;.*)?$', value: 'de/vbk/fv1' }
+        - { regex: '^(.*;)?DE-VBK:fv2(;.*)?$', value: 'de/vbk/fv2' }
+        - { regex: '^(.*;)?DE-VBK:fv3(;.*)?$', value: 'de/vbk/fv3' }
+    tags:
+        - { tag: 'railway:signal:distant', value: 'DE-VBK:fv' }
+        - { tag: 'railway:signal:distant:form', value: 'light' }
+
+  - description: Karlsruhe VBK Vorsignale
+    country: DE
+    icon: { default: 'de/vbk/fv0' }
+    tags:
+        - { tag: 'railway:signal:distant', values: ['DE-VBK:fv', 'DE-VBK:fv0'] }
+        - { tag: 'railway:signal:distant:form', value: 'sign' }
+
+  - description: Karlsruhe AVG Vorsignale
```

Comment:
I would expect this to be a Bahn Ubergang

Reply:
Oh yes, you're right!